src/rules/color-hex-length/README.md

```diff
@@ -0,0 +1,59 @@
+# color-hex-length
+
+Specify short or long notation for hex colors.
+
+```css
+    a { color: #fff }
+/**              ↑
+ * These hex colors */
+```
+
+## Options
+
+`string`: `"short"|"long"`
```

Comment:
I think `"always"|"never"` works well for whitespace rules, but I think we should stick with explicit options for everything else as, I think, it'll allow us to keep things more consistent i.e.

If we go down the `"always"|"never"` route for these types of rules, at what point do we stop using `"always"|"never"` and use explicit options instead? For example

* renaming something like `color-hex-case: "upper"|"lower"` to `color-hex-lowercase: "always"|"never"` kind of makes sense.
* but what about renaming something like `string-quotes: "single"|"lower"` to `string-quotes-single: "always"|"never"`. 
* and that, in turn, doesn't make sense when we get to rules with three explicit options like `url-quotes: "single"|"double"|"none"`.

I'd be keen to avoid this inconsistency by using explicit options for non-whitespace rules.

Secondly, I think I'd prefer to explicit over implicit e.g. `color-hex-lowercase: "never"` implies uppercase, `color-hex-short: "never"` implies long, and `string-quotes-single : "never"` implies double.

Thirdly, and unlike the whitespace rules, using `"always"|"never"` for these types of rules shows a preference for one options over another e.g. why `color-hex-short` and not `color-hex-long`, or why `color-hex-lowercase` and not `color-hex-uppercase` etc.

What do you think?